src/renderer/components/+add-cluster/add-cluster.tsx

```diff
@@ -71,8 +71,9 @@ export class AddCluster extends React.Component {
     if (value instanceof KubeConfig) {
       const context = value.currentContext;
       const isNew = userStore.newContexts.has(context);
+      const className = `${context} kube-context flex gaps align-center`
       return (
-        <div className="kube-context flex gaps align-center">
+        <div className={className}>
```

Comment:
We can select correct kube context in integration test by adding kubecontext name to `className` property.